include/motor-control/firmware/stepper_motor/pipette_motor_hardware.hpp

```diff
@@ -0,0 +1,52 @@
+#pragma once
```

Comment:
This definitely shouldn't be here - it's specific to pipettes so it should be in `include/pipettes` somewhere.